src/OpenFeature.Contrib.Providers.Flagd/Proto/csharp/Schema.cs

```diff
@@ -0,0 +1,2943 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: schema/v1/schema.proto
+// </auto-generated>
```

Comment:
I agree, having this in the .gitignore definitely makes sense! I will try to get the files being generated on demand by the build and will let you know if I manage to do it in this PR, or if i need to do some more research + an extra PR to set this up in a more tidy way

Reply:
alright, I think I managed to do it :) the Grpc client is now generated during the build, based on the .proto file in the imported submodule. @toddbaert do you know if in the build pipeline all git submodules are pulled when checking out the code? If not, i think this might need to be adapted to make the build + tests pass

Reply:
I will add this!

Reply:
I've done it.